src/main/java/eu/europa/ted/efx/interfaces/MarkupGenerator.java

```diff
@@ -68,9 +72,22 @@ public interface MarkupGenerator {
    */
   Markup composeFragmentDefinition(final String name, String number, Markup content);
 
+  /**
+   * Given a fragment name (identifier) and some pre-rendered content, this method
+   * returns the code
+   * that encapsulates it in the target template.
+   */
+  Markup composeFragmentDefinition(final String name, String number, Markup content, Set<String> parameters);
```

Comment:
This was merged before I could have a look at it, but I'd like to point out that adding a method to an **interface** is in general considered a breaking change: existing implementations of this interface will not build with the new version, they will need to add an implementation for the new method(s).
(For example, `MarkupGeneratorMock` had to be adapted in this project)

Reply:
Yes I know. You probably mean that this should be released as version 2 right? I am not sure yet when this will be released. But your point will be taken into account when it does. Maybe we will release this with version 2 of the SDK anyway. Thanks.